### PR TITLE
[PR from 3.x_doc] Update FasterRCNN doc with the right URL of R-50-FPN with BoundedIoULoss 

### PR DESCRIPTION
**Pull request by songyuc**
## Motivation

The original the URL of ["R-50-FPN BoundedIoULoss"](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn/README.md#different-regression-loss) do not point to the corresponding config. 

## Modification

Modified the URL of R-50-FPN with BoundedIoULoss.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.

### CODE REVIEW

**Comment by RangiLyu** [comment]
The lint failed, Please run `pip install pre-commit` and then run `pre-commit run --all-files` to fix the lint.

**Comment by songyuc** [comment]
> The lint failed, Please run `pip install pre-commit` and then run `pre-commit run --all-files` to fix the lint.

Now, I just check it by clicking my modified URL, then manually replacing "master" with "3.x" and refreshing the page.
If the config page is shown normally, I can confirm it will be right in the next release from 3.x to the coming master.

**Review by songyuc — COMMENTED** [review]
I commit again

File: configs/faster_rcnn/README.md [hunk 1]
```diff
@@ -34,12 +34,12 @@ State-of-the-art object detection networks depend on region proposal algorithms
...
+| R-50-FPN |    GIoULoss    |          |                |  37.6  |       [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py)       |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
+| R-50-FPN | BoundedIoULoss |          |                |  37.4  | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn/faster-rcnn_r50_fpn_bounded-iou_1x_coco.py) |     [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_bounded_iou_1x_coco-98ad993b.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_bounded_iou_1x_coco_20200505_160738.log.json)     |
```
Comment (RangiLyu) [inline configs/faster_rcnn/README.md:42]:
```suggestion
| R-50-FPN | BoundedIoULoss |          |                |  37.4  | [config](./faster-rcnn_r50_fpn_bounded-iou_1x_coco.py) |     [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_bounded_iou_1x_coco-98ad993b.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_bounded_iou_1x_coco_20200505_160738.log.json)     |
```

Comment (RangiLyu) [reply configs/faster_rcnn/README.md:42]: better to use a relative path

Comment (songyuc) [reply configs/faster_rcnn/README.md:42]: I will check for it, thank you!

**Review by RangiLyu — APPROVED** [review]

**Review by ZwwWayne — DISMISSED** [review]

**Comment by ZwwWayne** [comment]
Hi @songyuc ,
Thanks for your kind contribution. Would you like to fix the lint issue so that we could merge this PR?
The details can be found by clicking the item of lint check as shown below
image

**Comment by songyuc** [comment]
> Hi @songyuc , Thanks for your kind contribution. Would you like to fix the lint issue so that we could merge this PR? The details can be found by clicking the item of lint check as shown below image

Hi, @ZwwWayne! Thanks for your patient review. I have fixed the the lint issue.

**Review by RangiLyu — APPROVED** [review]

**Review by ZwwWayne — APPROVED** [review]

**Comment by OpenMMLab-Miao** [comment]
Hi @songyuc ！First of all, we want to express our gratitude for your significant PR in the MMDet  project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**